extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -100,10 +102,13 @@ public function getPages() {
 
 		$dbh = wfGetDB( DB_SLAVE );
 		( new \WikiaSQL() )
-			->SELECT_ALL()
+			->SELECT( "page.*, comments_index.*, IF(pp.props is NULL, page.page_id, REPLACE(REPLACE(REPLACE(pp.props, ';', ''), 'i', ''), ':', '')) as idx" )
```

Comment:
And actually actually, you don't seem to use the display order here (https://github.com/Wikia/pandora/pull/2225/files#diff-e5c435a43d88dbeabadc3185f2abe152R153) but instead number based on the biggest number already there, so you could leave off any special handling or renumbering altogether.